README.md

```diff
@@ -28,3 +28,5 @@ A collaborative list of AITU student's projects. To add your own project below,
 *List of projects of students that are studying telecommunication systems* 
 
 - [Alibek Mombekov/SpeechRecognition](https://github.com/olimeme/speechrecognition) – speech recognition system written on Python.
+## Cybersecurity
```

Comment:
- Rename "Cybersecurity" into "Cyber Security".
- Add "*List of projects of students that are studying cyber security*" after section name as in the other sections.
- Put this section before "Telecommunication Systems" section to follow the specified in "Navigation" section order.